docs/esp32/tutorial/peripheral_access.rst

```diff
@@ -42,3 +42,73 @@ for this:
 
 The MCPWM0 peripheral is in bit position 17 of the above two registers, hence
 the value of ``DPORT_PWM0_CLK_EN``.
+
+Synchronous access to pins directly via registers
+-------------------------------------------------
+
+.. code-block:: python3
```

Comment:
I suggest adding a short paragraph before the code example, to explain what it is doing.

Reply:
Added